framework/source/class/qx/util/Wheel.js

```diff
@@ -39,6 +39,11 @@ qx.Bootstrap.define("qx.util.Wheel", {
      */
     FACTOR: 1,
 
+    /**
+     * Is the Wheel actually a touchpad ?
+     * @internal
```

Comment:
Sigh. Leave it as it is, but it's wrong for all of them. If variables are not supposed to be touched from outside of a class, they should be private, i.e., with two leading underscores, and then they are "@internal" by default. I don't think it's appropriate to change all of them at this point, but that would really be the proper solution.